src/controllers/userController.js

```diff
@@ -21,10 +27,20 @@ export default class UserController {
 
     async addUser(req, res, next) {
         try {
-            await this.usersList.create(req.body);
+            const hash = bcrypt.hashSync(req.body.password, 10);
```

Comment:
@lenon777, why are you using the sync version of the method? The sync method can bring performance degradation.